extension/src/paymentHandler/refund-payment.handler.js

```diff
@@ -47,6 +47,12 @@ async function execute(paymentObject) {
             response.pspReference
           )
         )
+        actions.push(
+          pU.createChangeTransactionTimestampAction(
+            refundTransaction.id,
+            paymentObject.createdAt
```

Comment:
same as above

Reply:
Undo the timestamp assignment in extension module.
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/9e35ab7f3f1285abb7119ef3a73d820293b3fba2